e2e/go.mod

```diff
@@ -27,6 +27,7 @@ require (
 )
 
 require (
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerregistry/armcontainerregistry v1.3.0-beta.1 // indirect
 	github.com/Azure/go-autorest/autorest v0.11.29 // indirect
```

Comment:
I had to use the 1.3.0-beta.1 over the stable 1.2.0 in order to have access to `AnonymousPullEnabled` in the update params, which is a required field to be able to pull from the ACR (while we don't have oras login)